tests/models/whisper/test_modeling_whisper.py

```diff
@@ -547,10 +547,16 @@ def test_generate_language(self):
 
         # test language code
         model.generate(input_features, language="en")
-        # test tokenizer code
+        # test language token
         model.generate(input_features, language="<|en|>")
         # test language name
         model.generate(input_features, language="English")
+        # test language code list
+        model.generate(input_features, language=["en"] * input_features.shape[0])
+        # test language token list
+        model.generate(input_features, language=["<|en|>"] * input_features.shape[0])
+        # test language name list
+        model.generate(input_features, language=["English"] * input_features.shape[0])
```

Comment:
This is great!

Could you also test:
1. On this same test, checking that passing a list of languages that is not the same length as input_features does indeed raise an error ?
2. On a slow test, that you get the same results generating two samples with two languages at once vs generating each sample with one language at once ?

Reply:
Done!